static/images/fa-brands_telegram.svg

```diff
@@ -0,0 +1 @@
+<svg width="62" height="64" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill="#fff" d="M12 13h36v38H12z"/><path d="M31 1C13.875 1 0 14.875 0 32s13.875 31 31 31 31-13.875 31-31S48.125 1 31 1zm15.225 21.238-5.087 23.974c-.375 1.7-1.388 2.113-2.8 1.313l-7.75-5.712-3.738 3.6c-.413.412-.763.762-1.563.762l.55-7.887L40.2 25.313c.625-.55-.138-.863-.963-.313l-17.75 11.175-7.65-2.387c-1.662-.525-1.7-1.663.35-2.463L44.076 19.8c1.387-.5 2.6.337 2.15 2.438z" fill="#1D98DC"/></svg>
```

Comment:
I suggest removing `fill`, `height` and `width` attributes from this SVG, and specify `currentColor` for `fill` and use CSS to specify sizes. WDYT?